src/v3/test/integration/__snapshots__/authenticator-enroll-data-phone.test.tsx.snap

```diff
@@ -217,1286 +217,1308 @@ exports[`authenticator-enroll-data-phone should render form 1`] = `
-                              for="authenticator.phoneNumber"
-                            >
-                              Phone number
-                            </label>
                             <div
-                              class="MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-adornedStart emotion-45"
-                              dir="ltr"
+                              class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth emotion-45"
                             >
-                              <span
-                                class="MuiInputAdornment-root MuiInputAdornment-positionStart no-translate emotion-46"
-                              >
-                                <p
-                                  class="MuiTypography-root MuiTypography-body1 emotion-47"
+                              <label
+                                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-sizeMedium MuiInputLabel-outlined emotion-39"
+                                data-shrink="false"
+                                for="authenticator.phoneNumber"
+                                id="authenticator.phoneNumber-label"
+                              >
+                                <span>
+                                  Phone number
+                                </span>
+                              </label>
+                              <div
+                                class="MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedStart emotion-47"
+                                required="true"
+                              >
+                                <div
+                                  class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-outlined MuiInputAdornment-sizeMedium emotion-48"
                                 >
-                                  +1
-                                </p>
-                              </span>
-                              <input
-                                aria-invalid="false"
-                                autocomplete="tel-national"
-                                class="MuiInputBase-input MuiInputBase-inputAdornedStart emotion-48"
-                                data-se="authenticator.phoneNumber"
-                                id="authenticator.phoneNumber"
-                                inputmode="tel"
-                                name="authenticator.phoneNumber"
-                                type="tel"
-                              />
+                                  <span
+                                    class="notranslate"
+                                  >
+                                    ​
+                                  </span>
```

Comment:
MUI has some bug where a zero-width space was trying to be translated for `TextField` components. They fixed it by wrapping the space in a `<span class="notranslate" />` which appears here

https://github.com/mui/material-ui/pull/24563